pegasus/sites.v3/code.org/views/professional_learning_skinny_banner.haml

```diff
@@ -0,0 +1,23 @@
+- display_professional_learning_banners = true
```

Comment:
what's the purpose of this constant?

Reply:
This banner is used on a bunch of pages, so this'll make it easier to turn on and off each year instead of needing to manually add and remove it from each page.